codec.proto

```diff
@@ -9,44 +9,50 @@ import "google/protobuf/descriptor.proto";
 
 // used in Vote
 enum SignedMsgType {
-  option (gogoproto.goproto_enum_stringer)  = true;
-  option (gogoproto.goproto_enum_prefix)    = false;
+  option(gogoproto.goproto_enum_stringer) = true; option(gogoproto.goproto_enum_prefix) = false;
```

Comment:
weird that this is being pushed onto one line (same on line 24), is this just formatting?

Reply:
oh is it because there's no space between `options` and the `(gogoproto` anymore? Is this intended?

Reply:
I was testing `clang-format`, you can find command in readme :)